server/controllers/ArticleController.js

```diff
@@ -69,9 +74,20 @@ class ArticleController {
         ]
       })
       .then((result) => {
-        res.status(200).json({
+        const { rows, count } = result;
+        const pageCount = Math.ceil(count / limit);
+        if (+page > pageCount) {
+          return res.status(422).json({
+            errors: {
+              status: 'failure',
+              message: 'available number of page(s) exceeded'
+            }
+          });
+        }
+        return res.status(200).json({
           status: 'success',
-          articles: result.rows
+          availablePages: pageCount,
+          articles: rows
```

Comment:
In addition, you should also be returning the total no of articles, current page and num of articles on current page

Reply:
@mbilesanmi done